crypto-ballerina/build.gradle

```
@@ -19,6 +19,21 @@ import org.apache.tools.ant.taskdefs.condition.Os
 
 description = 'Ballerina - Ballerina Crypto'
 
+def packageName = "crypto"
+def packageOrg = "ballerina"
+def ballerinaConfigFile = new File("$project.projectDir/Ballerina.toml")
+def ballerinaDependencyFile = new File("$project.projectDir/Dependencies.toml")
+def artifactBallerinaDocs = file("$project.projectDir/build/docs_parent/")
+def artifactCacheParent = file("$project.projectDir/build/cache_parent/")
+def artifactLibParent = file("$project.projectDir/build/lib_parent/")
+def artifactCodeCoverageReport = file("$project.projectDir/target/cache/tests_cache/coverage/ballerina.exec")
+def tomlVersion = project.version.split("-")[0]
```

Check the dependent stdlib module version extracting in the updateToml task
```suggestion
def tomlVersion = project.version.replace("-SNAPSHOT", "")
```